backend/grant-service/src/main/kotlin/grant/rpc/GrantController.kt

```diff
@@ -58,6 +63,22 @@ class GrantController(
             ok(Unit)
         }
 
+        implement(Grants.transferApplication) {
+            val application =  applications.viewApplicationById(db, ctx.securityPrincipal.toActor(), request.applicationId)
+            if (application.first.grantRecipient.toString() != GrantRecipient.EXISTING_PROJECT_TYPE) {
```

Comment:
Move this check to the service and use `application.first.grantRecipient !is GrantRecipient.ExistingProject`